packstream/packstream-specification-1.md

```diff
@@ -809,15 +814,52 @@ DateTimeZoneId::Structure(
 )
 ```
 
-- The `seconds` are seconds since the adjusted [Unix epoch](https://en.wikipedia.org/wiki/Epoch_(computing)). This is not [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time).
+
 - The `tz_id` is an identifier for a specific time zone, such as `"Europe/Paris"`.
+- The `seconds` are the offset corresponding to the above timezone at the given point in time, added to the seconds 
+since the [Unix epoch](https://en.wikipedia.org/wiki/Epoch_(computing)).
```

Comment:
Similarly as above, I sadly don't understand this better than what we had before.